support-payment-api/src/main/scala/services/SwitchService.scala

```diff
@@ -46,21 +46,21 @@ case class FeatureSwitches(switches: FeatureSwitchesTypes)
 case class SwitchDetails(state: SwitchState)
 
 case class RecaptchaSwitchTypes(
-    enableRecaptchaBackend: SwitchDetails,
-    enableRecaptchaFrontend: SwitchDetails,
+    enableRecaptchaBackend: Option[SwitchDetails],
```

Comment:
I toyed with the idea of trying to do what support-frontend does and flattening it from `{ name: { state: 'On' }}` to just `{ name: 'On' }` but it seemed like too much change

Reply:
Yeah the flattened version is a lot easier to work with but maybe not worth it.

Reply:
Was also slightly beyond me in Scala 😀